server.js

```diff
@@ -32,8 +32,10 @@ module.exports = (opts, cb) => {
   const pathToWatch = opts.path || baseURL || '.'
   let ignoredPaths = [
     /[\/\\]\./,
```

Comment:
yes it is. Primarily chokidar-socket-emitter should watch html and js files. These are not topically in directories starting with dots.